core/src/main/scala/sttp/tapir/DecodeResult.scala

```diff
@@ -29,4 +29,6 @@ object DecodeResult {
         }
     }
   }
+
+  def apply[T](o: Option[T]): DecodeResult[T] = o.map(Value(_)).getOrElse(Missing)
```

Comment:
Tell me if it is ok, but I think transforming an Option into a DecodeResult will often make sens.
But maybe apply is too much maybe I should have name it "fromOption".
WDYT?

Reply:
Yes I think `fromOption` communicates the intent more clearly

Reply:
Done